test/helpers/transactions/InvokeScriptTransactionScreen.ts

```diff
@@ -0,0 +1,53 @@
+const Argument = (wrapped: WebdriverIO.Element) => ({
+  get type() {
+    return wrapped.findByTestId$('invokeArgumentType');
+  },
+
+  get value() {
+    return wrapped.findByTestId$('invokeArgumentValue');
+  },
+});
+
+const Payment = (wrapped: WebdriverIO.Element) => ({
+  get root() {
+    return wrapped;
+  },
+});
+
+export const InvokeScriptTransactionScreen = {
+  get root() {
+    return browser.$("[class*='transaction@']");
+  },
+
+  get paymentsTitle() {
+    return this.root.findByTestId$('invokeScriptPaymentsTitle');
+  },
+
+  get dApp() {
+    return this.root.findByTestId$('invokeScriptDApp');
+  },
+
+  get function() {
+    return this.root.findByTestId$('invokeScriptFunction');
+  },
+
+  async getArguments() {
+    try {
+      return await this.root
+        .findAllByTestId$('invokeArgument')
+        .map(it => Argument(it));
+    } catch (e) {
+      return [];
```

Comment:
Why do you ignore errors here and in `getPayments` below?

Reply:
Because find method waits for elements and throws error if elements aren't found.
And there is a case when i should check that there are no arguments or payments, so i need this try/catch block.

Reply:
May be use `queryAllBy*` then? It returns an empty array if there's no match
https://testing-library.com/docs/queries/about#types-of-queries

Reply:
Does that cause problems really? If it is, you can just wait for `root` first and then query payments

Reply:
done